chore: Remove current dark mode implementation

manishraj-003 opened the pull request:
Fixes #2999 
## Changes
- Replaced conditional theme logic with: `themeMode: ThemeMode.light`
- Set both `theme` and `darkTheme` to use `AppTheme.lightTheme`
- Ensures consistent UI and removes unexpected dark mode usage.
## Results
- App always stays in Light mode, even when device is in Dark mode
- Verified on:
  • Android Emulator (API 34)
  • System Dark Mode ON/OFF
  • App restart
- No visual/UI regressions introduced
## Screenshots / Recordings
The below recording shows the ui of the app when the system dark mode is 'ON' and 'OFF'.
https://github.com/user-attachments/assets/1aea9d84-fb25-4783-9cac-067bdc4920e5
**Checklist**: 
- [x] **No hard coding**: I have used values from `constants.dart` or localization files instead of hard-coded values.
- [x] **No end of file edits**: No modifications done at end of resource files.
- [x] **Code reformatting**: I have formatted the code using `dart format` or the IDE formatter.
- [x] **Code analysis**: My code passes checks run in `flutter analyze` and tests run in `flutter test`.

## Summary by Sourcery

Force the application to always use the light theme and clean up platform-specific artifacts.

Enhancements:
- Update Android night-mode styles so launch and normal themes use light variants instead of dark ones.

Chores:
- Remove obsolete Windows Flutter generated plugin registrant files from version control.
- Update dependency lockfile to reflect current package state.

manishraj-003 committed:
Remove current dark mode implementation

manishraj-003 committed:
chore: remove unintended Windows generated files

marcnause commented on macos/Flutter/GeneratedPluginRegistrant.swift:
Are the changes in this file necessary for this PR?

marcnause commented on pubspec.lock:
Are the changes in this file necessary for this PR?

marcnause commented on windows/flutter/generated_plugins.cmake:
Are the changes in this file necessary for this PR?

marcnause commented on android/app/src/main/res/values-night/styles.xml:
```diff
@@ -1,7 +1,7 @@
+    <style name="LaunchTheme" parent="@android:style/Theme.Light.NoTitleBar">
         <!-- Show a splash screen on the activity. Automatically removed when
              the Flutter engine draws its first frame -->
         <item name="android:windowBackground">@drawable/launch_background</item>
```
@copilot open a new pull request to apply changes based on [this feedback](https://github.com/fossasia/pslab-app/pull/3007#discussion_r2818433464)

marcnause committed:
Remove geolocator_windows from plugin list

marcnause committed:
Remove geolocator_apple plugin registration

Removed geolocator_apple plugin registration from the generated plugin registrant.

marcnause commented:
@sourcery-ai summary

marcnause committed:
removed folder for dark mode style

marcnause committed:
reverted changes in pubspec.lock

marcnause reviewed (approved):
Tested on my device
